File: demos/Simple Demo/Simple Demo/AboutViewController.m
//
//  AboutViewController.m
//  Simple Demo
//
//  Created by Muthu Arumugam on 10/6/15.
//  Copyright © 2015 Yieldmo, Inc. All rights reserved.
//

#import "AboutViewController.h"
@import Yieldmo;
@import AdSupport;

@interface AboutViewController () <UITextFieldDelegate>

@property (weak, nonatomic) IBOutlet UILabel * sdkVersionLabel;

@end

@implementation AboutViewController

- (void) viewDidLoad {
    [super viewDidLoad];

    self.sdkVersionLabel.text = [NSString stringWithFormat: @"v %@", [Yieldmo version]];
}

- (void) didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: demos/Swift/Simple Demo/Simple Demo/AboutViewController.swift
//
//  AboutViewController.swift
//  Simple Demo
//
//  Created by Muthu Arumugam on 10/7/15.
//  Copyright © 2015 Yieldmo, Inc. All rights reserved.
//

import UIKit

class AboutViewController: UIViewController {

    @IBOutlet weak var versionLabel: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()

        self.versionLabel.text = "SDK v\(YM.sharedYM().sdkVersion())"
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
